build.sbt

```
@@ -33,14 +32,12 @@ lazy val docs = project
 
 // General Settings
 lazy val commonSettings = Seq(
-  scalaVersion := "2.13.6",
-  crossScalaVersions := Seq(scalaVersion.value, "2.12.14"),
-  addCompilerPlugin("org.typelevel" % "kind-projector"     % "0.13.1" cross CrossVersion.full),
-  addCompilerPlugin("com.olegpy"   %% "better-monadic-for" % "0.3.1"),
+  scalaVersion := "3.0.1",
+  crossScalaVersions := Seq(scalaVersion.value, "2.12.13", "2.13.6"),
```

This is very dangerous and you shouldn't use this construction. I realize it's very common in a lot of Scala builds but we need to weed it out because it leads to insanity. Remember that `scalaVersion` is *changed* by any `++` or `+` usage, meaning that the value of `crossScalaVersions` will depend on the current Scala version, which is not at all what you want.

```suggestion
  crossScalaVersions := Seq("3.0.1", "2.12.13", "2.13.6"),
  scalaVersion := crossScalaVersions.value.last,
```

The above accomplishes the same thing without the associated problems.